soroban-sdk/test_snapshots/tests/auth/auth_30_deep_one_address_repeat/test.1.json

```diff
@@ -205,7 +205,7 @@
                 "contract": "CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M",
                 "key": {
                   "ledger_key_nonce": {
-                    "nonce": 2
+                    "nonce": "2"
```

Comment:
Did the type of this change or something?

Reply:
These must be the XDR->JSON conversion changes implemented by @leighmcculloch in rs-stellar-xdr recently.

Reply:
Yup. It's worth noting that the deserializer is backwards compatible and can still handle the original, but the write out of the same structures will result in strings for any greater than 32bit integer.